tests/data/test_data_references.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from seedpod_ground_risk.data import *
+
+
+class DataRefsTestCase(unittest.TestCase):
+
+    def test_aircraft_list_filepath(self):
+        aircraft_list = aircraft_list_filepath()
+        aircraft_fp_win = 'static_data\\aircraft_list.json'
+        aircraft_fp_lin = 'static_data/aircraft_list.json'
+        self.assertIn(aircraft_list, {aircraft_fp_lin, aircraft_fp_win})
```

Comment:
Since you're checking both platforms, you may as well use something like 
```python
import platform
...
if platform.system() == "Linux":
    # assert linux path
elif platform.system() == "Windows":
    # assert windows path
```